locale/surveyForm/da.yaml

```diff
@@ -0,0 +1,2 @@
+select:
+    default: Vælge et alternativ
```

Comment:
I don't think this is the correct translation. This sounds to me like infinitive (to do something), not imperative (as in when requesting that someone do something). It's an easy mistake to make when using machine translation since most(?) English verbs are the same for infinitive and imperative mood.